### ISSUE TITLE
Health probe creation for Azure Internal Load Balancer - to one Backend pool(VM) : aks-engine 0.67.0 & AKS 1.20.11

### ISSUE THREAD

**Issue by zohebs341** [Azure/aks-engine#5100]
Hi Team,

I've created an Azure Internal Load Balancer and added two standard VMs as a backend pool. I am trying to create health probes for this internal LB, but it's failing with the below error.

Failed to create probe 'tcp-15021'. Error: Not all Virtual Machines are indirectly referenced by the Load Balancer /subscriptions/de0e/resourceGroups/prod/providers/Microsoft.Network/loadBalancers/abc-internal-lb belong to the same availability set.

Basically, I am trying to access containers running in VMs from our Azure Stack AKS pods. For that reason, I've created internal LB and added a backend pool as the two VMs which are having containers up/running.

In order to use Axure internal LB, the VMs should have an availability set. In my case, the new VMs doesn't have availability sets.

**Comment by bridgetkromhout** [comment]
Hi, @zohebs341 - please open your issue in https://github.com/Azure/aks-engine-azurestack because AKS Engine is deprecated except for that Azure Stack use case. Thanks!
